prometheus/values.yaml

```
@@ -37,45 +30,49 @@ kube-state-metrics:
       cpu: "500m"
       memory: "2Gi"
 
-####### This block disables control plane components which are not reachable from within the Gardener data plane
+####### This block disabled not needed features
```

```suggestion
####### This block disables not needed features
```
"Disabled" or "disables"?